examples/single-system.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
```

Comment:
In fact, single-system.py and test.py were old testing scripts that should have been removed at a previous stage (I suspect I failed to git rm), the three 'mean-field...py' examples were the versions I intended to keep. Thanks for spotting this.

Reply:
Similarly 'tempo.py' shouldn't be present. This is a little concerning, because these are quite old example scripts and I remember tidying/removing the examples directory!